<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algoritmia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_agregar">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_Particulas">
          <property name="title">
           <string>Partículas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_id"/>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="spinBox_destinoY">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_origenX">
             <property name="text">
              <string>Origen(x):</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_destinoY">
             <property name="text">
              <string>Destino(Y)</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_green">
             <property name="text">
              <string>green:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_origenX">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_AgregarInicio">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Agregar al Inicio</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QSpinBox" name="spinBox_green">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_origenY">
             <property name="text">
              <string>Origen(Y)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="spinBox_destinoX">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="spinBox_red">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit_velocidad"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_velocidad">
             <property name="text">
              <string>Velocidad:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSpinBox" name="spinBox_blue">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_blue">
             <property name="text">
              <string>blue:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_id">
             <property name="text">
              <string>iD:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_destinoX">
             <property name="text">
              <string>Destino(X)</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_red">
             <property name="text">
              <string>red:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinBox_origenY">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_color">
             <property name="text">
              <string>Color:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_AgregarFinal">
             <property name="text">
              <string>Agregar al Final</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_Mostrar">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="salida_Particulas"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_tabla">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="6">
         <widget class="QTableWidget" name="table"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_IDParticula">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>ID Partícula</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_OrdenarID">
          <property name="text">
           <string>Ordenar por ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_OrdenarDistancia">
          <property name="text">
           <string>Ordenar por distancia</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButton_OrdenarVelocidad">
          <property name="text">
           <string>Ordenar por velocidad</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="pushButton_Buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="pushButton_mostrarTabla">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_particulas">
       <attribute name="title">
        <string>Partículas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_Dibujar">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_Limpiar">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
